### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Remove hardcoded items from tasks array
- Create a new folder module with files: functionalities and form
- Implement function for adding a new task
- Implement function for removing task
- Implement function for editing task descriptions
- Save data to local storage

**Latest changes:**

- Modify remove functionality to use index.js
- Add validations to form input and edit description

### CODE REVIEW

File: src/modules/functionality.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const removeList = (taskArray, fillTask, el) => {
+  const index = parseInt(el.children[0].children[0].getAttribute('id'), 10);
+  taskArray.splice(index, 1);
+  taskArray.forEach((entry, i) => {
+    entry.index = i;
+  });
+  fillTask();
+  el.remove();
+};
```
Comment (Lordkaito) [inline src/modules/functionality.js]: - [x] Please make sure to implement this functionality properly. Your function is defined, but you are not exporting it so it's not being used by your index.js file to delete tasks.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Yazino12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/functionality.js (hunk 1 above)
Comment (Yazino12) [reply src/modules/functionality.js]: Hi @Lordkaito, Thanks for the review, will implement that change right away 📝

**Review by Lordkaito — APPROVED** [review]
Hi @Yazino12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] [OPTIONAL] It would be good if, whenever you delete 2 or more items, you update the array of items on localStorage. Right now, if you delete 2 or more items and then reload the page, the last item deleted comes back, which should not happen, so take that into account for the upcoming milestones!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**